src/app/parcels/form/ParcelForm.tsx

```diff
@@ -200,7 +200,7 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
             client_id: (clientId || fields.clientId)!,
             packing_date: packingDate,
             packing_slot: fields.packingSlot,
-            voucher_number: fields.voucherNumber,
+            voucher_number: fields.voucherNumber ?? null,
```

Comment:
If the voucher number happens to be an empty string, we should store that as NULL in the database